docs/integrations-in-rancher/cloud-marketplace/aws-marketplace-payg-integration/troubleshooting.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Troubleshooting
+---
+
+## Troubleshooting
+
+This section contains information to help you troubleshoot issues when installing Rancher Prime PAYG.
+
+Jobs and Pods:
+
+Check that pods or jobs have status Running/Completed
+
+
+```
+kubectl get pods --all-namespaces
+```
+
+
+If a pod is not in Running state, you can debug into the root cause by running:
+
+Describe pod
+
+
+```
+kubectl describe pod <pod name> -n <namespaces>
+```
+
+Pod container logs
+
+
+```
+kubectl logs <pod name> -n <namespaces>
+```
+
+Describe job
+
+
+```
+kubectl describe job <job name> -n <namespaces>
+```
+
+Logs from the containers of pods of the job
+
+```
+kubectl logs -l job-name=<job name> -n <namespaces>
+```
+
+###Recovery from Failed Pods
+
+If any of the pods are not Running : 
```

Comment:
```suggestion
If any of the pods aren't running, check the rancher-cloud Pod:
```

If rancher-cloud Pod is just an example, suggest using `grep <pod name>` instead of `grep rancher-cloud`